shared/chat/header.tsx

```diff
@@ -31,36 +31,40 @@ type Props = {
   fullName?: string
 }
 
-const descColor = Styles.globalColors.black_50
-
-const descStyleMobile = {
-  color: descColor,
-  fontSize: 13,
-  lineHeight: 17,
-}
-const descStyleDesktop = {
-  fontSize: 13,
-  lineHeight: '17px',
-  wordBreak: 'break-all',
-} as const // approximates BodySmall since markdown does not support text type
-const descStyle = Container.isMobile ? descStyleMobile : descStyleDesktop
-
-const descStyleOverride = {
-  del: descStyle,
-  em: descStyle,
-  fence: descStyle,
-  inlineCode: descStyle,
-  kbfsPath: descStyle,
-  link: descStyle,
-  mailto: descStyle,
-  paragraph: descStyle,
-  preview: descStyle,
-  strong: descStyle,
-} as any
-
 const Header = (p: Props) => {
   const {desc, canEditDesc, isTeam, participants, fullName, channel, showActions, muted, username} = p
   const {onToggleInfoPanel, onToggleThreadSearch, unMuteConversation, onOpenFolder} = p
+  const isDarkMode = Styles.isDarkMode()
+
+  const descStyleOverride = React.useMemo(() => {
+    const descStyleMobile = {
+      color: Styles.globalColors.black_50,
+      fontSize: 13,
+      lineHeight: 17,
+    }
+    const descStyleDesktop = {
+      fontSize: 13,
+      lineHeight: 17,
```

Comment:
this needs `px`

Reply:
ok then  we need ti fix it further in markdown cause px isn't allowed in teh style and we convert it elsewhere

Reply:
updated